packages/asset-uploader/src/utils/format-asset.test.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { Asset, Location } from "@webstudio-is/prisma-client";
+
+const commonAsset: Asset = {
+  id: "sa-546",
+  name: "test",
+  location: Location.FS,
+  projectId: "id",
+  size: 2135,
+  format: "png",
+  createdAt: new Date(),
+  description: "",
+  meta: JSON.stringify({ width: 200, height: 200 }),
+};
+
+describe("getAssetPath", () => {
+  const OLD_ENV = process.env;
```

Comment:
I got confused for a second, we use SCREAMING_CASE only when we need to export constants like a config, not when its a local constant